package.json

```diff
@@ -2,7 +2,8 @@
   "name": "angularjs.org",
   "version": "0.0.0",
   "scripts": {
-    "build": "yarn && gulp && node scripts/build"
+    "build": "yarn && gulp && node scripts/build",
+    "build-local": "yarn && gulp && node scripts/build copy test"
```

Comment:
Is this any different than `build`? AFAICT, `copy test` are the default argument (when none are specified).

Reply:
Now that you mention it ...